code/_onclick/item_attack.dm

```diff
@@ -31,19 +31,26 @@ avoid code duplication. This includes items that may sometimes act as a standard
 
 // No comment
 /atom/proc/attackby(obj/item/W, mob/user, var/click_params)
+	user.client.interact_log += "([time_stamp()]) - Interacted with [src], using [W]. (Intent: [user.a_intent]|Location: [get_area(src)])"
```

Comment:
Add coordinates here. Add spaces next to the `|`, so `[thing] | [thing]`.

Reply:
I think coordinates will spam, no? Location should be enough

Reply:
Area's a bit vague imo.